README.md

```diff
@@ -2,7 +2,7 @@
 
 [![codecov](https://codecov.io/gh/AY2425S1-CS2103-F12-1/tp/graph/badge.svg?token=4P6LHV4HJ0)](https://codecov.io/gh/AY2425S1-CS2103-F12-1/tp)
 
-![appScreenshot.png](docs/images/appScreenshot.png)
+![appScreenshot.png](docs/images/Ui.png)
```

Comment:
Should we update `appScreenshot.png` to match the new `Ui.png` name?

Reply:
we should! thanks for picking this up :)

Reply:
changed :)